_studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer.cpp

```diff
@@ -98,18 +98,13 @@ Packer::~Packer()
 /****************************************************************************************************/
 // VA linux packer implementation
 /****************************************************************************************************/
-PackerVA::PackerVA(VideoAccelerator * va)
-    : Packer(va)
-{
-}
 
-Status PackerVA::GetStatusReport(void * , size_t )
+void PackerVA::PackPicParams(const H265DecoderFrame *pCurrentFrame, TaskSupplier_H265 *supplier)
 {
-    return UMC_OK;
-}
+    H265DecoderFrameInfo const* pSliceInfo = pCurrentFrame->GetAU();
```

Comment:
shouldn't you check input pointers?

Reply:
_PackPicParams_ is private function. It makes sense to add this check to _PackAU_ which is public (interface).

Reply:
Add checks to _PackPicParams_